integrations/pendo/src/script.raw.js

```diff
@@ -21,5 +21,20 @@
         y.src = `${cdn}/agent/static/` + apiKey + '/pendo.js';
         z = e.getElementsByTagName(n)[0];
         z.parentNode.insertBefore(y, z);
+        y.onload = function () {
+            pendo.initialize({
```

Comment:
Based on their documentation it looks like you may be able to call initialize with a special id to track visitor as anonymous:

> Call pendo.initialize() when your application first loads and then pass a special string ('VISITOR-UNIQUE-ID') to the Visitor ID. This generate a [Pendo anonymous Visitor ID](https://support.pendo.io/hc/en-us/articles/360032202751).
https://support.pendo.io/hc/en-us/articles/360031862272-Install-Pendo-on-a-single-page-web-application

Something like this
```suggestion
            pendo.initialize({
                visitor: {
                    id: 'VISITOR-UNIQUE-ID',
                 }
            });
```

have you tried this?

There also could be some side effect to tracking visitors as anonymous and never identifying them afterwards but not sure.

Reply:
@spastorelli I heard back from their support. This is what they said:

"I am not mega knowledgeable but the part you have right now doesn’t do anything, it’s just the declaration of what the Pendo snippet should do/behave etc. “pendo.initialize” is what actually runs a function to load the Pendo snippet on the page/app/etc so you need to have it in there. It doesn’t matter if you don’t have a login, this function is what loads Pendo in the thing, you can skip the visitor/account data, what will happen is that you will only get anonymous visitors/usage. Let me know if this helps!"

So I am thinking we can get away with `pendo.initialize({})`

Reply:
Wdyt?

Reply:
Yep sounds good. I was only suggesting as this is something they mention in their docs. It's not very clear though but yeah if it works with empty object that's good.